docs/_webCode/js/downloads.js

```diff
@@ -4,9 +4,8 @@
 // (like in your pom.xml).
 // E.g.: https://nexus.betonquest.org/repository/betonquest/
 
-document$.subscribe(() => {
-
-  const nexusUrl = "${REPOSITORY_URL}";
+document$.subscribe(async () => {
+  const nexusUrl = "https://nexus.betonquest.org/repository/betonquest/";
```

Comment:
Why inline the nexus URL instead of using the placeholder?

Reply:
accident from testing, i do a roll back of this